hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -1872,9 +1872,9 @@ public void deleteVolume(String volume) throws IOException {
       if (!ozAdmins.contains(ProtobufRpcEngine.Server.
           getRemoteUser().getUserName())
           && !ozAdmins.contains(OZONE_ADMINISTRATORS_WILDCARD)) {
-        LOG.error("Only admin users are authorized to create " +
+        LOG.error("Only admin users are authorized to list " +
             "Ozone volumes.");
-        throw new OMException("Only admin users are authorized to create " +
+        throw new OMException("Only admin users are authorized to list " +
             "Ozone volumes.", ResultCodes.PERMISSION_DENIED);
```

Comment:
Nice find.  Can you please also extract the message to a variable to avoid duplication?